scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -53,6 +54,16 @@ export function AggregateCtrl($scope, desks, workspaces, preferencesService, sto
     };
     this.activeFilterTags = {};
 
+    if (appConfig.features.hideCreatePackage) {
+        availableFileTypes.forEach((file) => {
+            if (file.type !== 'composite' && file.type !== 'highlight-pack') {
+                this.fileTypes.push(file);
+            }
+        });
+    } else {
+        this.fileTypes = availableFileTypes;
+    }
```

Comment:
imo this would be easier to read using `.filter`
```ts
this.fileTypes = availableFileTypes.filter(
    (fileType) => (
        fileType.type !== 'composite' ||
        fileType.type !== 'highlight-pack' ||
        appConfig.features.hideCreatePackage !== false
    ));
```